### PR TITLE
Added SD and .editorconfig

### PR DESCRIPTION
**Pull request by SteveWallace**
Updated with proper quoting and added basic .editorconfig to stop prettier from double quoting

### CODE REVIEW

File: scrapers.js [hunk 1]
```diff
@@ -61,7 +63,8 @@ let scrapers = [
     }
   },
   {
-    url: 'https://opendata.arcgis.com/datasets/8840fd8ac1314f5188e6cf98b525321c_0.csv',
+    url:
+      'https://opendata.arcgis.com/datasets/8840fd8ac1314f5188e6cf98b525321c_0.csv',
```
Comment (lazd) [inline scrapers.js]: Still dropping these guys down to the next line, can we prevent that?

Comment (SteveWallace) [reply scrapers.js]: oh, sorry didn't notice that

Yeah, give me one minute.

File: scrapers.js [hunk 2]
```diff
@@ -302,19 +320,35 @@ let scrapers = [
       let $ = await fetch.page(this.url);
 
       let cases = 0;
-      cases += parse.number($('p:contains("Number of PHL positives")').first().text().split(': ')[1]);
-      cases += parse.number($('p:contains("Number of commercial lab positives")').first().text().split(': ')[1]);
+      cases += parse.number(
+        $('p:contains("Number of PHL positives")')
+          .first()
+          .text()
+          .split(': ')[1]
```
Comment (lazd) [inline scrapers.js]: Chaining is much clear like this, so let's keep it the way it is in your PR.

Can we make prettier do this automatically before commits?

File: scrapers.js [hunk 3]
```diff
@@ -200,11 +213,10 @@ let scrapers = [
         if (string === 'None') {
           return 0;
         }
-        if (matches = string.match(/(\d+) of (\d+)/)) {
+        if ((matches = string.match(/(\d+) of (\d+)/))) {
           // Return the high number
           return parse.number(matches[2]);
-        }
-        else {
+        } else {
```
Comment (lazd) [inline scrapers.js]: Definitely want `else` on a newline.

**Comment by lazd** [comment]
Thanks so much for working through the weird coding style bits here. In the haste of setting this project up, I didn't bother with the usual, so getting this PR in with Prettier behaving right will definitely help ease some growing pains.

File: scrapers.js (hunk 2 above)
Comment (SteveWallace) [reply scrapers.js]: on saves, yes. not sure on commits? I'll verify

**Comment by lazd** [comment]
Also, no need to re-send the PR once you get it back up to date with master, just force push to your branch.

**Comment by SteveWallace** [comment]
@lazd updated. Turns out the prettier team is very opinionated about where an else/elseif goes. They won't add the option for it to be on a new line. Workaround is to add a stub comment between the bracket and else. I did that for the one that's in scraper.js.

**Comment by lazd** [comment]
Excellent contribution. Thank you!

**Pull request merged**